UnityProject/Assets/Scripts/Learning/ProtipObjectTypes/ProtipObjectOnEnterRadius.cs

```diff
@@ -24,10 +24,6 @@ private void CheckForNearbyItems()
 			var possibleTargets = Physics2D.OverlapCircleAll(gameObject.AssumedWorldPosServer(), SearchRadius, MaskToCheck);
 			foreach (var target in possibleTargets)
 			{
-				if (Application.isEditor)
-				{
-					Debug.DrawLine(gameObject.AssumedWorldPosServer(), target.gameObject.AssumedWorldPosServer(), Color.green, 8f);
-				}
 				if(gameObject == target.gameObject) continue;
 				if (MatrixManager.Linecast(gameObject.AssumedWorldPosServer(), LayerTypeSelection.Walls,
```

Comment:
If you want to debug what's causing 100 length recasts, You can put a debugging here to check the distance

Reply:
I already did and I've already complained on discord about it being stupid. There are zero issues with the already very small range I have for it but for some reason staging has that error. I'm hoping that changing this number once can fix it.